src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -430,6 +430,17 @@ private static String getHashOfName(String name) {
         return Long.toString(Math.abs((long) name.hashCode()));
     }
 
+    public static String getAnonEmailWithoutHash(FeedbackParticipantType type) {
+        String anonName = getAnonNameWithoutHash(type);
+        return anonName + "@@" + anonName + ".com";
+    }
+
+    public static String getAnonNameWithoutHash(FeedbackParticipantType type) {
```

Comment:
Avoid having references to the hash in new code since the hash is slated to be removed completely from the code base.